src/terraform-provider-paperspace/resource_network.go

```diff
@@ -52,14 +52,15 @@ func resourceNetworkCreate(d *schema.ResourceData, m interface{}) error {
 
 	createNamedNetworkParams := CreateTeamNamedNetworkParams{
 		Name:     name,
-		RegionId: regionId,
+		RegionId: regionID,
 	}
 
 	if err := paperspaceClient.CreateTeamNamedNetwork(teamID, createNamedNetworkParams); err != nil {
 		return fmt.Errorf("Error creating private network: %s", err)
 	}
 
-	return resource.Retry(d.Timeout(schema.TimeoutCreate), func() *resource.RetryError {
+	timeout, _ := time.ParseDuration("2m")
```

Comment:
1. you could update `schema.TimeoutCreate` on the schema itself to `2m` rather than overriding the value inline here, i think
1. separately, _technically_ i wouldn't parse this timeout anew each time, but rather do this outside of the calling scope (like in global scope), if it's not going to be on the schema